shared/selene/util/email/templates/password_change.html

```diff
@@ -0,0 +1,32 @@
+<div style="background-color: #e6e8ea; width: 100%; height:100%;">
+    <img
+            style="display: block; height: 60px; margin: auto; padding-top: 32px; color: white; font-family: Arial,sans-serif; font-size: 18px"
+            src="https://mycroftai.imgus11.com/public//dd014aa691ea98bb0a4b194b19bb2a0e.png?r=537470560"
+            alt="MYCROFT AI"
+    />
+    <div style="
+            background-color: white;
+            border-radius: 4px;
+            box-shadow: 0 8px 16px 0 rgba(0,0,0,0.40);
+            color: #2c3e50;
+            font-family: Helvetica Neue,Helvetica,arial,sans-serif;
+            height: 400px;
+            margin: 16px auto;
+            width: 480px;
+        "
+    >
+        <div style="padding: 32px">
+            <p>Greetings,</p>
+            <p style="margin-top: 16px;">
+                The password for the account with this email address on account.mycroft.ai has been changed.  If you
+                updated your password, you can ignore this message.
```

Comment:
Maybe add something like: 
"If you did not change your password, please contact us immediately support@mycroft.ai"

Reply:
Done